kennel:
  ros__parameters:
    real_time_factor: 2.0
    sim_time_update_period_ms: 2
    map_yaml_filename: /home/docker-dev/wombat/kennel/maps/hospital_section.yaml
    #map_yaml_filename: ""
    map_frame_id: ground_truth
    map_topic_name: ground_truth_map
    robots:
      - "my_robot"

my_robot:
  ros__parameters:
    mobile_base:
      update_period_ms: 5
      ground_truth_map_topic_name: ground_truth_map
      #ground_truth_map_topic_name: ""
      slam_update_period_ms: 100
      control_topic_name: cmd_vel
      control_msg_lifespan_ms: 200
      start_pose: [1.0, 1.0, 0.0]
